### PR TITLE
made project compatible with latest versions and reset project to basics

### PR DESCRIPTION
**Pull request by hydraadra112**

### CODE REVIEW

File: src/main.tsx
Comment (computerscience-person) [inline src/main.tsx:1]: wtf did you do here?

File: src/index.css
Comment (computerscience-person) [inline src/index.css:1]: Please bring the tailwind stuff back

File: src/App.tsx
Comment (computerscience-person) [inline src/App.tsx:1]: I'm seeing that you use semicolons at the end of every line, should we enforce that to all future code? Or just mix and match? Or use a code formatter like Prettier?

**Review by computerscience-person — CHANGES_REQUESTED** [review]
Just look over lang sa mga comments, thanks.

File: src/App.tsx
Comment (hydraadra112) [reply src/App.tsx:1]: may prettier nana ssob

File: src/index.css
Comment (hydraadra112) [reply src/index.css:1]: why ? i removed it para maka start gid ta from scratch and indi ma confuse ya iba ta nga upod

File: src/main.tsx
Comment (hydraadra112) [reply src/main.tsx:1]: reset the project to basic header 1 hello world

File: src/index.css
Comment (computerscience-person) [reply src/index.css:1]: Needed for tailwind stuff (I thought requirement ni Sir)

Comment (hydraadra112) [reply src/index.css:1]: yea but we can do our own tailwind stuff

File: src/App.tsx
Comment (computerscience-person) [reply src/App.tsx:1]: I mean, like add prettier and eslint sa dev dependencies...

File: src/main.tsx
Comment (hydraadra112) [reply src/main.tsx:1]: actually, nothing. i just ran the line `npm install react react-dom @types/react @types/react-dom`, then it resolved all errors

File: src/App.tsx
Comment (hydraadra112) [reply src/App.tsx:1]: just saw my code again. in App.tsx. where specifically ko ni ibutang? extension mani dan ang prettier

File: src/index.css
Comment (computerscience-person) [reply src/index.css:1]: Can you elaborate?

Comment (hydraadra112) [reply src/index.css:1]: i will be adding the headers or dependencies import but not the code

File: src/main.tsx
Comment (computerscience-person) [reply src/main.tsx:1]: Awah lol, sometimes ga work sakon sometimes wala... LGTM

File: src/index.css
Comment (hydraadra112) [reply src/index.css:1]: > Can you elaborate?

we'll be styling our own components for more control and so we can follow the planned design of aika and reycel

Comment (computerscience-person) [reply src/index.css:1]: Tailwind isn't opinionated man, so I don't think ma affect man na sa implementation ka design. Let's put a pin on this for later discussion nalang anay.

File: src/App.tsx
Comment (computerscience-person) [reply src/App.tsx:1]: IG ako nalang masetup, since it's not straightforward abi...

**Comment by computerscience-person** [comment]
Rebased!

**Pull request closed without merging**